File: src/Domain/Entities/Artist.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace projects.Domain.Entities;
public class Artist : BaseAuditableEntity
{
    public Artist()
    {
        Albums = new HashSet<Album>();
    }

    public int Id { get; set; }
    public string Name { get; set; }
    public string MusicType { get; set; }
    public byte[] Photo { get; set; }
    public ICollection<Album> Albums { get; set; }
}

File: src/WebUI/ClientApp/src/app/models/artist.ts
export class Artist {
  id: number | undefined;
  name?: string;
  musicType? : string;
  photo?: string;

  constructor(data: any = null) {
    if (data) {
      this.id = data.id;
      this.name = data.name;
      this.musicType = data.musicType
      this.photo = data.photo;
    }
  }
}
